pyslackersweb/contexts.py

```diff
@@ -8,9 +8,14 @@
 
 async def apscheduler(app: web.Application) -> AsyncGenerator[None, None]:
     app["scheduler"] = app["website_app"]["scheduler"] = AsyncIOScheduler()
-    app["scheduler"].start()
+
+    if not app["SKIP_SCHEDULER"]:
+        app["scheduler"].start()
+
     yield
-    app["scheduler"].shutdown()
+
+    if not app["SKIP_SCHEDULER"]:
```

Comment:
I think it reads nicely to change this to:

```python
if app["scheduler"].running:
    app["scheduler"].shutdown()
```

Which removes one more thing looking at the `app`'s state: https://apscheduler.readthedocs.io/en/latest/modules/schedulers/base.html#apscheduler.schedulers.base.BaseScheduler.running